### PR TITLE
Stickdom basic files added in

### PR DESCRIPTION
**Pull request by jwong28**
- Stickdom and first phaser game has been added in, the first phaser game is all in 1 file (part1.html) 
- Stickdom converted into multiple files for states within phaser 3. 
- Images have also been updated such as the player sprite is now a stickman. 
- Bullets has been changed into arrows.  
- Readme has been updated slightly, will need more adjustments. 
- Changelog has been added. 
- Note: The switch from 1 file to multiple files has led to changes in code.

### CODE REVIEW

**Comment by AryanJ-NYC** [comment]
More descriptive PR name, please ("Jefferson" isn't descriptive enough, sorry).

Use bullets for readability.

**Comment by AryanJ-NYC** [comment]
Additionally, in future, PRs with 274K+ lines of changes will likely be rejected.

File: Stickdom/Index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!doctype.htlml>
+<html> 
+<head> 
+    <script src="phaser.js"></script>
```
Comment (AryanJ-NYC) [inline Stickdom/Index.html]: Use the `phaser.min.js` file and delete `phaser.js` (preferred) or vice versa.

File: Stickdom/part1.html [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+        }
+    },
+    scene: {
+        preload: preload,
+        create: create,
+        update: update
+    }
+};
+
+    var game = new Phaser.Game(config);
```
Comment (AryanJ-NYC) [inline Stickdom/part1.html]: Why is this indented? Everything all the way down is badly indented as well.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+- 3/21/2018: Hello Phaser has been uploaded. For the future, a basic version of my game will be added onto this repo.
+- 3/26/2018: First Phaser game is added. Adapted from the Phaser 3 tutorial.
+- 4/102018: Stickdom first commit. Copied first phaser game
```
Comment (AryanJ-NYC) [inline CHANGELOG.md]: You forgot a slash.

**Comment by jwong28** [comment]
As of 4/16/2018
- CDN implemented so phaser files removed
- Part 1 removed since I converted it to the js files

File: Stickdom/level1.js [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+        this.anims.create({
+            key: 'right',
+            frames: this.anims.generateFrameNumbers('player', { start: 0, end: 9 }),
+            frameRate: 10,
+            repeat: -1
+        });
+
+        this.anims.create({
+            key: 'faceLeft',
+            // frames: [ { key: 'player', frame: 0 } ],
```
[the remark this answers is not in the document]
Comment (AryanJ-NYC) [reply Stickdom/level1.js]: This (line 70 in `level1.js`) is still not fixed. There are other similar comments in the same file.

@jwong28 Please remove.

**Comment by AryanJ-NYC** [comment]
The comments still aren't removed @jwong28

**Pull request merged**